source/docs/software/commandbased/commands.rst

```diff
@@ -1,67 +1,277 @@
+Command Properties
+------------------
+
+In addition to the four lifecycle methods described above, each ``Command`` also has two properties, the value of which is defined by the ``runsWhenDisabled`` and ``getInterruptBehavior`` methods respectively. In most cases, the default value is enough.
+
+getRequirements
+^^^^^^^^^^^^^^^
+
+Each command should declare any subsystems it controls as requirements. This backs the scheduler's resource management mechanism, ensuring that no more than one command requires a given subsystem at the same time. This prevents situations such as two different pieces of code attempting to set the same motor controller to different output values.
+
+Most command factories and constructors provided by the library have a ``requirements`` vararg (Java) / initializer list (C++) parameter.
+
+Declaring requirements is done by overriding the ``getRequirements()`` method in the relevant command class, by calling ``addRequirements()``, or by using the ``requirements`` vararg (Java) / initializer list (C++) parameter at the end of the parameter list of most command constructors and factories in the library.
+
+As a rule, command compositions require the union of the requirements of their component commands.
+
+runsWhenDisabled
+^^^^^^^^^^^^^^^^
+
+The ``runsWhenDisabled()`` method returns a ``boolean``/``bool`` specifying whether the command may run when the robot is disabled. With the default of returning ``false``, the command will be canceled when the robot is disabled and attempts to schedule it will do nothing. Returning ``true`` will allow the command to run and be scheduled when the robot is disabled.
+
+.. important:: Except for Addressable LEDs, hardware outputs are disabled when the robot is disabled, regardless of ``runsWhenDisabled()``!
+
+This property can be set either by overriding the ``runsWhenDisabled()`` method in the relevant command class, or by using the :ref:`docs/software/commandbased/decorators:ignoringDisable` decorator.
+
+As a rule, command compositions may run when disabled if all their component commands set ``runsWhenDisabled`` as ``true``.
+
+getInterruptionBehavior
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``getInterruptionBehavior()`` method (`Java <https://github.wpilib.org/allwpilib/docs/beta/java/edu/wpi/first/wpilibj2/command/Command.html#getInterruptionBehavior()>`__, `C++ <https://github.wpilib.org/allwpilib/docs/beta/cpp/classfrc2_1_1_command.html#ab1e027e86fc5c9132914ca566a9845a8>`__) defines what happens if another command sharing a requirement is scheduled while this one is running. In the default behavior, ``kCancelSelf``, the current command will be canceled and the incoming command will be scheduled successfully. If ``kCancelIncoming`` is returned, the incoming command's scheduling will be aborted and this command will continue running.
```

Comment:
I think it's worth saying explicitly that kCancelIncoming means that a command can't be interrupted.

Reply:
A command with `kCancelIncoming` can still be canceled, if that's what you're referring to. Interrupt behavior only defines what happens when another command with shared requirements is scheduled.